### DISCUSSION TITLE
Option to include the client that generated a posting (e.g. Apollo, RIF)

### DISCUSSION THREAD

**Discussion by phhowe17** [Serene-Arc/bulk-downloader-for-reddit#871] [Q&A]
Is there a way to log the end client app that generated a post.  I'd like to understand what clients a community is using.

**Comment by Serene-Arc** [comment]
Is this information included on Reddit? We don't have access to anything that isn't  already on the site. If you can't do it through the web browser, we can't do it through the BDFR.
